### PR TITLE
Support multiple exporters via OTLP_EXPORTER config

### LINKED ISSUE #496: Implement OTEL_EXPORTER env var
How an end-user can configure instrumentation agent to e.g. send spans to Zipkin and metrics to Prometheus? As of now only one exporter can be configured and some of them do not support metrics at all.

### PR DESCRIPTION
**Pull request by pavolloffay**
Signed-off-by: Pavol Loffay <ploffay@redhat.com>

Resolves #496 

PS: my first PR to this repo, I might have done smth wrong.

### CODE REVIEW

**Comment by iNikem** [comment]
@pavolloffay please run `./gradlew spotlessApply`

**Comment by pavolloffay** [comment]
@iNikem I have updated the PR

File: README.md [hunk 1]
```diff
@@ -56,7 +56,15 @@ Note: These parameter names are very likely to change over time, so please check
...
 behavior you may find.
 
-#### Jaeger exporter
+#### Exporters
+
+The following configuration properties are common to all exporters:
+
+| System property          | Environment variable     | Purpose                                                                                            |
+|--------------------------|--------------------------|----------------------------------------------------------------------------------------------------|
+| otel.exporter            | OTEL_EXPORTER=           | To select exporter e.g. `otlp,jaeger`. Defaults to `otlp`                                          |
```
Comment (iNikem) [inline README.md:65]: Still one extra `=`  :)

**Review by iNikem — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (pavolloffay) [reply README.md]: yay, removed.

**Review by trask — APPROVED** [review]
Thanks!

**Comment by malafeev** [comment]
am I right that next values for  OTEL_EXPORTER are supported: "otlp_span", "otlp_metric"? (https://github.com/open-telemetry/opentelemetry-specification/blob/2088381f57004be1de4fa16e0c2adeea04cf2ce2/specification/sdk-environment-variables.md#exporter-selection)

so if I specify `"otlp_span"` then metrics will not be enabled?

**Comment by trask** [comment]
Hey @malafeev, `otlp_span` and `otlp_metric` values are not supported yet. If you are interested in those, can you open an issue for adding support for them?

**Comment by malafeev** [comment]
created https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/1250

**Pull request merged**